### ISSUE TITLE
Peer Review - Alexander Coulson

### ISSUE THREAD

**Issue by AlexanderCoulson** [RaniaS605/Group1_Assessment#3]
Positives:
- Very informative and well-explained assignment based on a relevant topic in today's world. 
- Also has a very good explanation of the different models related to your topic.

Negatives:
- There is a lot of information simply thrown at the reader without much structure or organisation. The use of sub-titles and paragraphing would be of benefit as it would be easier on the eye for many readers

In summary, a very good read and very relevant to today's society and our worlds political structure
